site/src/Librecores/ProjectRepoBundle/Entity/ProjectClassifier.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+namespace Librecores\ProjectRepoBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * ProjectClassifier
+ *
+ * @ORM\Table(name="project_classifier")
+ * @ORM\Entity(repositoryClass="Librecores\ProjectRepoBundle\Repository\ProjectClassifierRepository")
+ */
+class ProjectClassifier
+{
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="id", type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="parent_id", type="integer")
+     */
+    private $parentId;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="categories", type="text")
+     */
+    private $categories;
+
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="parentUser_id", type="integer")
+     */
+    private $parentUserId;
+
+    /**
+     * @var \DateTime
+     *
+     * @ORM\Column(name="created_at", type="datetime")
+     */
+    private $createdAt;
+
+    /**
+     * @var \DateTime
+     *
+     * @ORM\Column(name="updated_at", type="datetime")
+     */
+    private $updatedAt;
```

Comment:
will you really update these entries, as opposed to deleting them and assigning new ones?